components/cache/adapters/redis_adapter.rst

```diff
@@ -62,15 +62,16 @@ helper method allows creating and configuring the Redis client class instance us
     );
 
 The DSN can specify either an IP/host (and an optional port) or a socket path, as well as a
-password and a database index.
+password and a database index. To enable the transport level security for connections, the scheme `redis` must be
```

Comment:
Please 2 backticks, same on the next line, thanks

Reply:
Done :+1:

Reply:
Done :+1: